website/docs/telemetry.md

```diff
@@ -60,6 +60,10 @@ Explicit user action is required (see below) to change an `undecided` state to `
 Telemetry reporting is always disabled for [embedded FerretDB](https://pkg.go.dev/github.com/FerretDB/FerretDB/ferretdb)
 and can't be configured.
 
+:::info
+Despite the autogenerated message by `mongosh` on telemetry collection, please note that no data will be shared with MongoDB Inc.
```

Comment:
```suggestion
Despite the autogenerated message by `mongosh` regarding MongoDB's free cloud-based monitoring service, please note that no data will ever be shared with MongoDB Inc.
```

Reply:
They don't ever refer to their service as telemetry, so it's slightly confusing to read. Hence the suggestion. Also, it connects to the `test` collection when you connect to your instance.

Reply:
Great suggestion! Thanks 🙌🏽